cluster/charts/rook-ceph/templates/NOTES.txt

```diff
@@ -1,5 +1,18 @@
 The Rook Operator has been installed. Check its status by running:
   kubectl --namespace {{ .Release.Namespace }} get pods -l "app=rook-ceph-operator"
 
-Visit https://rook.io/docs/rook/master for instructions on how
-to create & configure Rook clusters
+Visit https://rook.io/docs/rook/master for instructions on how to create and configure Rook clusters
+
+Note: You cannot just create a CephCluster resource, you need to also create a namespace and
+provide your own RBAC Roles and RoleBinding resources for the cluster. The Rook Operator
```

Comment:
They aren't really the "users own" RBAC roles, they need to create RBAC resources provided by Rook.
To me this makes it sound like the users need to write their own RBAC resources for Rook, could you try to rephrase this a bit to make it more sound like they use the existing RBAC and modify it accordingly?